db/migrate/20170519181110_add_description_to_collections.rb

```diff
@@ -0,0 +1,6 @@
+class AddDescriptionToCollections < ActiveRecord::Migration
+  def change
+    add_column :collections, :description, :text, default: ""
```

Comment:
Check "the bad" section of this blog post for details, https://www.braintreepayments.com/blog/safe-operations-for-high-volume-postgresql/